R/CoNGA.R

```diff
@@ -165,3 +165,64 @@ SeuratToCoNGA <- function(seuratObj,
                                path = R.utils::getAbsolutePath(paste0(seuratToCongaDir, "/GEX.h5"), mustWork = FALSE),
                                overwrite = TRUE)
 }
+
+#' @title Plot Diversity
+#'
+#' @description Plot a diversity profile for each library in the data
+#' @param conga_clones_file The output clones file from CoNGA (named clones_file.txt).
+#' @param OutputFile The output file containing the data used for generating diversity profiles.
+#' @param order1 The minimum order for calculating the generalized Simpson entropy.
+#' @param order2 The maximum order for calculating the generalized Simpson entropy.
+#' @param organism 'human' or 'rhesus'
+#' @examples
+#' \dontrun{
+#'   PlotDiversity(conga_clones_file = "./clones_file.txt",
+#'                 OutputFile = "./diversity_output.csv", 
+#'                 order1 = 1, order2 = 200)
+#' }
+#' @export
+
+PlotDiversity <- function(conga_clones_file = "./clones_file.txt",
```

Comment:
this is picky, but can you make OutputFile lower case? and do you need to set a default value for it? the latter isnt really a problem per se, but in general it seems less common to do this